pylint/checkers/utils.py

```diff
@@ -842,3 +842,41 @@ def is_registered_in_singledispatch_function(node):
             return decorated_with(func_def, singledispatch_qnames)
 
     return False
+
+
+def splitlines(value, keepends=False):
+    """Splitlines only on universal newlines for consistency across python2
```

Comment:
I think this is too heavy-weight for our purpose. Also, I'm not too keen on keeping compatibility between Python 2 and 3, we are going to drop support for the former in one-two years max, so I would not want to have code tailored for the former in particular. Why can't we use the builtin ``.splitlines`` here?